mne/preprocessing/ssp.py

```diff
@@ -25,6 +25,14 @@ def _safe_del_key(dict_, key):
     if key in dict_:
         del dict_[key]
 
+def _deprecate_average(_average):
+    average = False if _average is None else _average
+    if _average is None:
```

Comment:
This will never be True after the line above. Put the line above this one after this check

Reply:
(up to you if you want to change it)